test/src/test/java/hudson/slaves/NodeCanTakeTaskTest.java

```diff
@@ -96,13 +94,15 @@ public void becauseNodeIsBusy() throws Exception {
         project.setAssignedNode(slave);
         project.setConcurrentBuild(true);
         project.getBuildersList().add(new SleepBuilder(Long.MAX_VALUE));
-        project.scheduleBuild2(0).waitForStart(); // consume the one executor
+        FreeStyleBuild build = project.scheduleBuild2(0).waitForStart(); // consume the one executor
         project.scheduleBuild2(0); // now try to reschedule
         Queue.Item item;
         while ((item = r.jenkins.getQueue().getItem(project)) == null || !item.isBuildable()) {
             Thread.sleep(100);
         }
         assertEquals(hudson.model.Messages.Queue_WaitingForNextAvailableExecutorOn(slave.getDisplayName()), item.getWhy());
+        build.doStop();
+        r.assertBuildStatus(Result.ABORTED, r.waitForCompletion(build));
```

Comment:
Fixes a Windows flakiness bug.

Reply:
Not sure it is fully legit here, because it is rather a hackaround, But whatever